woodwork/logical_types.py

```diff
@@ -216,7 +217,7 @@ class BooleanNullable(LogicalType):
     primary_dtype = "boolean"
 
     def transform(self, series, null_invalid_values=False):
-        series = _replace_nans(series)
+        series = _replace_nans(series, BooleanNullable.primary_dtype)
```

Comment:
Is there a reason for using `BooleanNullable.primary_dtype` instead of `self.primary_dtype` here and in the other methods?

Reply:
It was just preference but I like `self.primary_type` better, thanks!